### ISSUE TITLE
Rust lifetime colors are incorrect

### ISSUE THREAD

**Issue by jkcclemens** [akamud/vscode-theme-onedark#22]
Atom
![Atom](http://i.imgur.com/qT5Ob5H.png)

VS Code
![VS Code](http://i.imgur.com/HGaMj4R.png)

See line 19:

```rust
fn entry<'a>(&self, _: &CommandInfo, matches: &ArgMatches<'a>) {
```

The `'a` is the lifetime, which is colored incorrectly.

**Comment by akamud** [comment]
Thanks for reporting. Can you provide a code sample that I can paste on my editor to fix this issue?

**Comment by jkcclemens** [comment]
Did you want something [like this](https://gist.github.com/3f8dce9658c173653151e262c444a427)?

**Comment by akamud** [comment]
Yes! Thank you! I'll report back when I release a new version.

**Comment by akamud** [comment]
This should now be fixed on `0.2.11`, please let me know if you find anything else.
